docs/java/workflows.md

```diff
@@ -162,29 +159,32 @@ Then, one of the methods
 (depending on which Workflow type has been started) annotated with `@WorkflowMethod` can be invoked.
 As soon as this method returns, the Workflow execution is considered as completed.
 
-While Workflow execution is active, it can receive signals and query requests. Note that you can 
-send query requests to workflows after their have completed execution as well.
+Workflow methods annotated with `@QueryMethod` and `@SignalMethod` can be invoked during Workflow execution.
+Note that methods annotated with `@QueryMethod` can be invoked even when a Workflow is in the `Completed` 
+state.
```

Comment:
I want to see each sentence on its own line, makes it way easier to review changes.
Does your editor have a max width that causes these to wrap ?

Reply:
probably, i will fix it. sorry